lib/conventions.js

```diff
@@ -80,16 +76,14 @@ function assign(target, source) {
 }
 
 /**
- * A number of attributes are boolean attributes.
- * The presence of a boolean attribute on an element represents the `true` value,
+ * A number of attributes are boolean attributes. The presence of a boolean attribute on an element represents the `true` value,
  * and the absence of the attribute represents the `false` value.
  *
- * If the attribute is present, its value must either be the empty string
- * or a value that is an ASCII case-insensitive match for the attribute's canonical name,
- * with no leading or trailing whitespace.
+ * If the attribute is present, its value must either be the empty string or a value that is an ASCII case-insensitive match for
+ * the attribute's canonical name, with no leading or trailing whitespace.
  *
- * Note: The values `"true"` and `"false"` are not allowed on boolean attributes.
- * To represent a `false` value, the attribute has to be omitted altogether.
+ * Note: The values `"true"` and `"false"` are not allowed on boolean attributes. To represent a `false` value, the attribute has
+ * to be omitted altogether.
```

Comment:
This was a byproduct of the other plugin as well. I have reverted all changes made by the other plugin, and formatted again from the scratch with the homer0 plugin.

Reply:
Not sure if there was some manual modification in there, but I just took the code for that from `master`, ran the plugin, and the only change was this:

```diff
- * If the attribute is present, its value must either be the empty string
- * or a value that is an ASCII case-insensitive match for the attribute's canonical name,
- * with no leading or trailing whitespace.
+ * If the attribute is present, its value must either be the empty string or a value that is an ASCII case-insensitive match for
+ * the attribute's canonical name,
+ * with no leading or trailing whitespace.
```

Because the default `printWidth` is `130`, so there was a lot of room in there for more words; but check that it respects the comma:


https://github.com/xmldom/xmldom/assets/6949310/44360328-4428-41fc-95b6-22ee491a2360

(Ignore the error on the actual JS code, that's because I took that snippet and placed in a temp file)
(I ran it with v6)